operator/internal/manifests/var.go

```diff
@@ -97,6 +97,8 @@ const (
 	httpTLSDir = "/var/run/tls/http"
 	// grpcTLSDir is the path that is mounted from the secret for TLS
 	grpcTLSDir = "/var/run/tls/grpc"
+	// tenantMTLSDir is the path that is mounted from the configmaps for mTLS
+	tenantMTLSDir = "/var/run/tls/tenants"
```

Comment:
Do we need an extra directory here? Can't we use the same directory/mount as for `oidc.issuerCAPath`?

Reply:
I might not be seeing something but I don't think `oidc.issuerCAPath` is ever mounted anywhere, AFAIC the value is just taken as is.

Reply:
Will be fixed by https://github.com/grafana/loki/pull/9972